doc/GettingStarted.md

```diff
@@ -1,31 +1,9 @@
 ---
-uid: Overview.Extensions
+uid: GettingStarted
```

Comment:
@lukeblevins were there any references to the Overview.Extensions id? If so, they need to be updated

Reply:
Handled in 1434dca3da8a42881f1158bc2e5edb5e87b94d9f